packages/leap/lib/src/leap_world.dart

```diff
@@ -7,30 +7,30 @@ import 'package:leap/leap.dart';
 /// and all of the [PhysicalEntity] components.
 ///
 /// Any [PhysicalEntity] added anywhere in the [LeapGame] component tree
-/// will automatically be part of the world via [physicals]
+/// will automatically be part of the world via [physicals].
 class LeapWorld extends PositionComponent with HasGameRef<LeapGame> {
```

Comment:
We should maybe re-name this to `LeapPhysicsComponent` so it's not confused with the new Flame `CameraComponent.world`

Reply:
On second thought, maybe this should extend `World` since conceptually it is. the same thing (the object the represents the game world). In the case of Flame's `World` it enables the Camera system, and in the case of `LeapWorld` it enables the collision and physics system, but in both cases we want ALL components which are part of the game's "world" to be children of the world object, so combining them might make sense.